indico/web/client/styles/design_system/README.txt

```diff
@@ -0,0 +1,114 @@
+Indico Design System
+====================
+
+
+This file is part of Indico.
+Copyright (C) 2002 - 2023 CERN
+
+Indico is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see the LICENSE file for more
+details.
+
```

Comment:
not needed in a random README (also because this would not be caught by our tool to update headers)